package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "pcln-design-system",
-  "version": "1.0.0-6",
```

Comment:
For PRs we shouldn't touch the version. This PR will probably be merged before others, but in some cases, other PRs could get merged in and this would be out of sync.

Currently we bump the version after merging to master, and do a publish immediately. Once we move out of the beta versions, we'll use `npm version` to manage versions and add git tags.